packages/shared/src/net/error.ts

```diff
@@ -3,6 +3,8 @@ import { AxiosError } from "axios";
 // https://nodejs.org/docs/latest-v18.x/api/errors.html#common-system-errors
 export type NodeNetworkError = "ECONNREFUSED" | "ECONNRESET";
 
+export type NodeResponseError = "ERR_BAD_RESPONSE";
```

Comment:
This is an Axios code, not Node - [reference of Node errors](https://nodejs.org/docs/latest-v18.x/api/errors.html#common-system-errors).

Reply:
adding it to the axios error list instead